robottelo/ui/login.py

```diff
@@ -38,8 +39,20 @@ def logout(self):
         self.wait_for_ajax()
 
     def is_logged(self):
-        """Checks whether an user is logged"""
-        if self.wait_until_element(locators["login.gravatar"]):
-            return True
-        else:
-            return False
+        """Checks whether user is logged by validating a session cookie"""
+        cookies = dict(
+            (cookie['name'], cookie['value'])
+            for cookie in self.browser.get_cookies()
+        )
+        # construct the link to the Dashboard web page
+        url_root = get_server_url() + '/dashboard'
+
+        response = requests.get(
+            url_root,
+            verify=False,
+            allow_redirects=False,
+            cookies=cookies
+        )
+        response.raise_for_status()
+        return response.status_code != 302 \
```

Comment:
Using `\` for line continuation is not a good practice, please use `()` instead like:

```python
        return (
            response.status_code != 302 or
            not response.headers['location'].endswith('/login')
        )
```

or 

```python
        return (response.status_code != 302 or
                not response.headers['location'].endswith('/login'))
```